src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute is pointing to a SCSS file (`main.scss`). HTML cannot directly use SCSS files; they need to be compiled into CSS first. Please ensure you link to the compiled CSS file instead.